frontend/components/Contribute.vue

```diff
@@ -374,7 +374,7 @@ export default {
   }
 
   .contribute-option-list {
-    font: Bold 12px/15px Proxima Nova;
```

Comment:
I didn't know this worked! What does Apx/Ypx mean? Is it supported by all browsers. @bomiku